c7n/resources/appelb.py

```diff
@@ -102,7 +102,7 @@ class ConfigAppElb(ConfigSource):
     def load_resource(self, item):
         resource = super(ConfigAppElb, self).load_resource(item)
         resource['Tags'] = [{u'Key': t['key'], u'Value': t['value']}
-          for t in json.loads(item['Tags'])]
+          for t in json.loads(item['supplementaryConfiguration']['Tags'])]
```

Comment:
What's the difference between this and the root `tags` (lowercase) key

Reply:
nothing significant.. primarily format - ones a list ones in canonical tag form (array of dicts), ones a k/v dictionary.